web-app/src/pages/for-employers/for-employers.tsx

```diff
@@ -236,151 +392,654 @@ function ForEmployers() {
-				<SectionTitle>
-					<T.heading1 />
-				</SectionTitle>
+			<ContentWrapper>
+				<Banner>
+					<Row>
+						<Col xs={12} md={6}>
+							<BannerContent>
+								<div>
+									<BannerNormalContent>
+										Hiring made{" "}
+									</BannerNormalContent>
+									<BannerHighlightedContent>
+										easier
+									</BannerHighlightedContent>
+									,<br />
+									<BannerHighlightedContent>
+										faster
+									</BannerHighlightedContent>
+									<BannerNormalContent>
+										, and{" "}
+									</BannerNormalContent>{" "}
+									<BannerHighlightedContent>
+										affordable
+									</BannerHighlightedContent>
+									<br />
+									<BannerNormalContent>
+										with
+									</BannerNormalContent>{" "}
+									<BannerVizeContent>
+										Vize
+										<VizeBackgroundEffect />
+									</BannerVizeContent>
+								</div>
+								<div>
+									<BannerSubTitle>
+										{" "}
+										Post jobs and we'll rank ad filter the
+										right factory workers in
+										<br />
+										Tijuana for your factory
+									</BannerSubTitle>
+								</div>
+								<BannerButtonContainer>
+									<LinkButton
+										$primary
+										to={
+											urlGenerators.queryRoutes
+												.submitSalaryData
+										}
+									>
+										GET STARTED
+									</LinkButton>
+								</BannerButtonContainer>
+							</BannerContent>
+						</Col>
+						<Col xs={12} md={6}>
+							<BannerImage
+								src={bannerImage}
+								alt="employer-banner"
+							></BannerImage>
+						</Col>
+					</Row>
+				</Banner>
 
-				<ProblemPoint>
-					<T.card1part1 /> <Bold>90% - 120%</Bold>
-					<T.card1part2 />{" "}
-					<Bold>
-						<T.card1part3 />
-					</Bold>
-				</ProblemPoint>
-
-				<ProblemPoint>
-					<T.card2part1 />{" "}
-					<Bold>
-						<T.card2part2 />
-					</Bold>
-				</ProblemPoint>
-				<ProblemPoint>
-					<T.card3part1 />{" "}
-					<Bold>
-						<T.card3part2 />
-					</Bold>
-				</ProblemPoint>
-			</SectionContainer>
-
-			<SectionContainer style={{ backgroundColor: "white" }}>
-				<SectionTitle>
-					<T.heading2 />
-				</SectionTitle>
-				<SolutionContainer>
-					<div>
-						<SubsectionTitle>
-							<T.recruitingHeading />
-						</SubsectionTitle>
-						<P>
-							<T.recruitingText />
-						</P>
-					</div>
-					<T.img.exampleJobPost
-						renderer={(t) => (
-							<img
-								className="img-responsive"
-								src={t.src}
-								alt={t.alt}
+				<FeaturesWrapper>
+					<Row>
+						<Col xs={12} md={4}>
+							<CardContent>
+								<CardImageWRapper
+									src={saveMoneyImage}
+									alt="save-money"
+								/>
+								<CardTitle>Save Money</CardTitle>
+								<div>
+									Get two months of free and unlimited job
+									posts by signing up today.You can hire
+									workforce you need easier, faster, and more
+									affordable by reaching over 3000 factory
+									workers on our site.
+								</div>
```

Comment:
Use a component named BenefitCardDescription with the appropriate styling instead of using div